tests/console/yast2_proxy.pm

```diff
@@ -80,22 +80,26 @@ sub run {
 
     # if firewall is enabled, then send_key alt-p, else move to page http ports
     unless ($is_still_susefirewall2) {
-        if (check_screen 'yast2_proxy_firewall_enabled') {
+        if (check_screen 'yast2_proxy_firewall_enabled', 10) {
             send_key 'alt-p';
             wait_still_screen 3;
+            # now we have a new popup to workaround for question about firewall settings
+            if (check_screen 'yast2_proxy_squid_cannot-open-interface') {
+                send_key 'alt-y';
+            }
```

Comment:
You already have experience with better ways, multi-tag `assert_screen` with `match_has_tag`. I don't see a reason why we should not use that here

Reply:
this part is quite old, of course I can re-work this now

Reply:
checked this again, I don't think it is better to keep this because we don't need to check more than 2 needles here:
http://e13.suse.de/tests/4551#step/yast2_proxy/12